packages/MarkdownEditor-Tests.package/MarkdownCommentBlockTests.class/instance/testMismatchOverlappingOpeningAndClosingTag.st

```diff
@@ -0,0 +1,5 @@
+tests - matching
+testMismatchOverlappingOpeningAndClosingTag
+
+	self deny: (MarkdownCommentBlock matches: '<!-->').
```

Comment:
after reading the spec, I think that the rules
1. Up to 3 indentation
2. Begins with `<!--`

are sufficient. It differs for inline comments though, where `<!-->` and `<!--->` and `<!-- -- -->` are forbidden.
So I think these tests are wrong and you can remove some lines in `matches:`. But maybe I just misread the spec...

Reply:
Hm seems like you are right. I think it's confusing that they have different rules abut trying it with https://spec.commonmark.org/dingus/ gives the result you described.